### PR TITLE
Replaced TestRPC with Ganache on every docs page but one

### PR DESCRIPTION
**Pull request by bmmpxf**

### CODE REVIEW

**Comment by bmmpxf** [comment]
The "but one" page is handled in a separate PR:

https://github.com/trufflesuite/trufflesuite.com/pull/15

File: public/docs/getting_started/client.md [hunk 1]
```diff
@@ -4,11 +4,24 @@ There are many Ethereum clients to choose from. We recommend different clients d
 
 ## When developing
 
-### Truffle Develop
+### Ganache
+
+We recommend [Ganache](/ganache), a one-click GUI blockchain you can use for development. It is designed to be interoperable with Truffle. It processes transactions instantly instead of waiting for the default block time -- so you can test that your code works quickly -- and it tells you immediately when your smart contracts run into errors. It also makes a great client for automated testing, and Truffle knows how to use its special features to speed up test runtime by almost 90% compared to other clients.
```
Comment (tcoulter) [inline public/docs/getting_started/client.md]: I want to edit the sell here to differentiate Ganache from our other chains (Ganache is all about seeing, personalization, and speed -- and likely more. We'll tighten this up as Ganache matures):  

> We recommend [Ganache](/ganache), a personal blockchain for Ethereum development that runs on your desktop. Part of the Truffle suite, Ganache simplifies dapp development by placing your contracts and transactions front and center. Using Ganache you can quickly see how your application affects the blockchain, and introspect details like your accounts, balances, contract creations and gas costs. You can also fine tune Ganache's advanced mining controls to better suit your needs. Ganache is available for Windows, Mac and Linux, and you can [download it now](/ganache).

File: public/docs/getting_started/client.md [hunk 2]
```diff
@@ -4,11 +4,24 @@ There are many Ethereum clients to choose from. We recommend different clients d
...
+candy maple cake sugar pudding cream honey rich smooth crumble sweet treat
+```
+
+This mnemonic can be changed to be randomly generated, or you can input your own.
+
+
+### Truffle Develop
 
-Because Truffle Develop is built into Truffle, no external installation is required. To run Truffle Develop, type the following into a terminal:
+We also recommend using Truffle Develop, a development blockchain built directly into Truffle. It has many of the same features as Ganache, but in addition is built into Truffle, with no external installation required. To run Truffle Develop, type the following into a terminal:
```
Comment (tcoulter) [inline public/docs/getting_started/client.md]: You say "built into truffle" / "built directly into Truffle" twice. We should change this. We also shouldn't compare Truffle Develop to Ganache, in that Truffle Develop as no UI, and if we say it's "like Ganache" they may be expecting something entirely different. 

I'd sell Truffle Develop like this: 

> We also recommend using Truffle Develop, a development blockchain built directly into Truffle. Truffle Develop helps you set up an integrated blockchain environment with a single command, no installation required. Run Truffle Develop by typing the following into a terminal:

File: public/docs/getting_started/client.md [hunk 3]
```diff
@@ -24,9 +37,9 @@ Once launched, Truffle Develop will provide you with a console you can use to ru
 
 To read more about interacting with the console, please see the [Using the Console](/docs/getting_started/console) section.
 
-### TestRPC
+### Ganache CLI
 
-You can also use [EthereumJS TestRPC](https://github.com/ethereumjs/testrpc). It's a separate command-line blockchain you can use for both development and automated testing, and works well with Truffle. Check out its [README](https://github.com/ethereumjs/testrpc/blob/master/README.md) and documentation for more information.
+Ganache has a command-line interface, called [Ganache CLI](https://github.com/trufflesuite/ganache-cli/), with similar functionality to Ganache. This is useful for those who want to use Ganache but aren't working in a graphical environment. Ganache CLI is [available separately]((https://github.com/trufflesuite/ganache-cli/).
```
Comment (tcoulter) [inline public/docs/getting_started/client.md]: Given the above changes, I'd reword this to pump the CLI use cases (i.e., formerly TestRPC use cases). Additionally, we don't want to say "similar functionality to Ganache" because that makes them sound like they're separate products (technically they are, but in people's heads they should remain the same, with all the same assumptions). 

I'd do:

> Ganache also has a command-line interface for those who aren't working from a graphical environment. Great for automated testing and continuous integration environments, Ganache CLI runs headless and can be configured to serve all your development needs. Ganache CLI processes transactions instantly instead of waiting for the default block time, so you can test that your code works quickly. It also tells you immediately when your smart contracts run into errors, and integrates directly with Truffle to reduce test runtime up to 90% compared to other clients. [Learn more.](https://github.com/trufflesuite/ganache-cli/)

File: public/docs/getting_started/client.md [hunk 4]
```diff
@@ -37,7 +50,7 @@ There are many official and unofficial Ethereum clients available for you to use
 * Parity: [https://github.com/paritytech/parity](https://github.com/paritytech/parity)
 * More: [https://www.ethereum.org/cli](https://www.ethereum.org/cli)
 
-These are full client implementations that include mining, networking, blocks and transaction processing. You should use these clients after you've sufficiently tested your dapp with Truffle Develop or the TestRPC and you're ready to deploy to your desired Ethereum network. Truffle can deploy to these clients without any extra configuration.
+These are full client implementations that include mining, networking, blocks and transaction processing. You should use these clients after you've sufficiently tested your dapp with Ganache or Truffle Develop and you're ready to deploy to your desired Ethereum network. Truffle can deploy to these clients without any extra configuration.
```
Comment (tcoulter) [inline public/docs/getting_started/client.md]: Noticed that this is slightly wrong: 

> Truffle can deploy to these clients without any extra configuration.

Technically you do need to add configuration steps to deploy to these networks, but you don't need _more than_ what's needed for Ganache or Truffle develop. To remove confusion, we should probably remove or reword that sentence.

**Comment by bmmpxf** [comment]
This is excellent, precisely the kind of wording that I was looking for. Thanks!
